pkg/tekton-pipelines/tekton-pipelines.go

```diff
@@ -150,6 +158,27 @@ func reconcileTektonPipelinesFuncs(pipelines []pipeline.Pipeline) []common.Recon
 	return funcs
 }
 
+func reconcilePipelineRunsFuncs(pipelineRuns []pipeline.PipelineRun) []common.ReconcileFunc {
```

Comment:
How does deploying PipelineRuns with the operator work? The ISO URL is still needed, right?

Reply:
This functionality is implemented to show user, that pipelineRun for win11 needs some more configuration (e.g. the user and group settings for modify-windows-iso task). When TTO deploys this pipelineRun, it creates a pipelineRun which is in pending state and newer runs (the pending state is defined in pipelineRun manifest). So user can copy it, update it and insert the URL.

Reply:
Can't that be part of the documentation / examples like we did for the windows10 pipeline? Always having a PipelineRun in pending state looks unclean to me. Also what happens if the users deletes this PipelineRun, is it recreated?